tests/cache-cmd.earth

```diff
@@ -0,0 +1,20 @@
+VERSION 0.6

```

Comment:
I wonder if we should have tests or examples based on user feedback, like caching Elixir’s Dialyzer or Maven.

Reply:
I would rather keep the testing as simple as possible (without over simplifying the requirements).

and then additionally we could create a real-world example (under `./examples`) to ensure it solves our use-cases.

Reply:
Yeah I think some Earthfiles in ./examples is fine too

Reply:
Good idea. I can easily add examples for Maven and NPM within this PR. I'll have to look into the Elixir's Dialyzer example a bit more but if its straight forward I can include that as well.